src/OpenFOAM/meshes/primitiveMesh/PrimitivePatch/PrimitivePatch.H

```diff
@@ -117,7 +117,7 @@ private:
 
         //- Reference to global list of points
         PointField points_;
-
+        mutable gpuField<PointType>* gpuPointsPtr_;
```

Comment:
This pointer cleared in the destructor. For this purpose use deleteDemandDrivenData macro.